auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextEmptyPathTest.java

```diff
@@ -22,25 +22,30 @@
 import org.junit.Test;
 
 import javax.net.ssl.SSLContext;
+import javax.validation.constraints.AssertTrue;
+
 import java.security.NoSuchAlgorithmException;
 import java.security.Security;
 
 /**
  * Test that default SSLContext provider will throw an exception when configured file path is empty
  */
 public class DefaultSSLContextEmptyPathTest {
-
-    @Test(expected = IllegalArgumentException.class)
+    
```

Comment:
@leahgarbri  Just a minor, this will result in checkstyle plugin's failure. 
When you try to run `mvn clean install` on the project, it will say:

```
Line has trailing spaces. [RegexpSingleline]
```

This means you have to remove the empty spaces from the line 34.